pkg/cmd/template/schema_test.go

```diff
@@ -933,6 +933,162 @@ foo: #@ data.values.foo
 	})
 }
 
+func TestSchema_Overlay_multiple_schema_files(t *testing.T) {
+	opts := cmdtpl.NewOptions()
+	opts.SchemaEnabled = true
+
+	t.Run("when additional schema file is scoped to root library", func(t *testing.T) {
+		schemaYAML1 := `#@schema/match data_values=True
+---
+db_conn:
+- hostname: ""
+`
+
+		schemaYAML2 := `#@ load("@ytt:overlay", "overlay")
+#@schema/match data_values=True
+#@overlay/match by=overlay.all
```

Comment:
something to think about... why do we need `#@overlay/match by=overlay.all` on schemas but not on data/values?

Reply:
Good catch.. we don't need this annotation. This must've been a copy/paste mistake. Fixed.